PublishParasoftResults/task.json

```diff
@@ -50,7 +50,7 @@
             "label": "Root path to Parasoft tool/Java installation",
             "defaultValue": "",
             "required": false,
-            "helpMarkDown": "Absolute location of the root path of a Parasoft tool or a standalone Java installation (Parasoft tools contain a Java installation). When found at the specified location, Java is used to optimize the efficiency of report processing. If not specified or found, report processing will be much slower."
+            "helpMarkDown": "Absolute location of the root path of a Parasoft tool or a standalone Java installation (Parasoft tools contain a Java installation). When found at the specified location, Java is used to optimize the efficiency of report processing. If not specified or found, report processing will be much slower and it is possible to introduce the memory issues when processing large report."
```

Comment:
The screenshot is ![image](https://github.com/parasoft/parasoft-findings-vsts/assets/109122571/d82365e3-19e9-45c2-9f40-19ef15b1f07c)